pkgs/tools/misc/uutils-coreutils/default.nix

```diff
@@ -2,19 +2,24 @@
 , Security
 }:
 
-rustPlatform.buildRustPackage {
-  name = "uutils-coreutils-2019-05-03";
+let
+  pname = "uutils-coreutils";
+  version = "0.0.2";
+
   src = fetchFromGitHub {
     owner = "uutils";
     repo = "coreutils";
-    rev = "036dd812958ace22d973acf7b370f58072049dac";
-    sha256 = "0d9w3iiphhsk7l5l34682wayp90rgq5a3d94l3qdvhcqkfmpg727";
+    rev = version;
+    sha256 = "0rlvwmyk8c597x1krvlyyqhk1m7g2pz2rkwsam8yj3ksb670g53j";
   };
+in
```

Comment:
Please remove that let in.

Reply:
Is there a reason for that other than your personal preference?

Reply:
This is how we do it in nixpkgs. Doing let in is usually only done when fetching extra sources.

Reply:
> Is there a reason for that other than your personal preference?

There is no good reason to use a let in here. rec is shorter, cleaner, easier to understand even for non functional programmers and more commonly used which makes it easier to do treewide changes against it. Please change it.